apps/system/utils/utils_stackmonitor.c

```diff
@@ -205,6 +203,7 @@ static int stkmon_read_proc(FAR struct dirent *entryp, FAR void *arg)
 
 static void *stackmonitor_daemon(void *arg)
 {
+	struct stkmon_save_s *terminated_infos;
```

Comment:
almost same opinion as my previous comment.
This function do not use 'terminated_infos', then it had better alloc & free in there.